### DISCUSSION TITLE
Can't get device template from repository sync

### DISCUSSION THREAD

**Discussion by Nadia00485** [opendcim/openDCIM#1565] [Q&A]
<img width="503" alt="picture" src="https://github.com/user-attachments/assets/d9ed4392-c054-40cb-ad49-6355cdb7ec5f">

Hi, I'm encountering an issue when trying to pull from the master branch. After I click 'Pull from master', nothing happens, and there is no feedback or error message. Could someone help me troubleshoot this?

**Comment by dbmello75** [comment]
Hi, do you still need help??? 
I went through same problem last week.
